modules/webrtc/library_godot_webrtc.js

```diff
@@ -129,66 +129,66 @@ const GodotRTCDataChannel = {
 		return 0;
 	},
 
-	godot_js_rtc_datachannel_is_ordered: function(p_id) {
+	godot_js_rtc_datachannel_is_ordered: function (p_id) {
 		return IDHandler.get_prop(p_id, 'ordered', true);
 	},
 
-	godot_js_rtc_datachannel_id_get: function(p_id) {
+	godot_js_rtc_datachannel_id_get: function (p_id) {
 		return IDHandler.get_prop(p_id, 'id', 65535);
 	},
 
-	godot_js_rtc_datachannel_max_packet_lifetime_get: function(p_id) {
+	godot_js_rtc_datachannel_max_packet_lifetime_get: function (p_id) {
 		const ref = IDHandler.get(p_id);
 		if (!ref) {
 			return 65535;
 		}
 		if (ref['maxPacketLifeTime'] !== undefined) {
 			return ref['maxPacketLifeTime'];
-		} else if (ref['maxRetransmitTime'] !== undefined) {
+		} if (ref['maxRetransmitTime'] !== undefined) {
```

Comment:
That looks bogus, or is it a JS feature?

Reply:
Well it's valid actually since the previous block is a `return`, but then I guess it should be on a newline.

Reply:
I think this is just a bad rule that recognize:
```
if () {
  return
} esle if() {
  return
}
```
and remove the `else` because IIRC sequential ifs are faster then if/elses:
https://eslint.org/docs/rules/no-else-return
I'll see if I can force it to a newline, but maybe we should just remove the rule, I don't like it too much.

Reply:
There's only a few cases where it ends as `} if ... {` so the newline could possibly be added manually.